apps/plea/tests/test_validators.py

```diff
@@ -40,19 +42,25 @@ def test_urn_invalid_database(self):
 
     def test_is_valid_urn_format(self):
         good_urns = [
-            "06/AA/0000000/00",
+            "06/AA/12345/99",
+            "06/AA/0012345/99",
             "06/bb/1234567/12",
             "06/JJ/50563/14",
             "06/JJ/50534/14",
+            "06AB/12345/99",
+            "06AB12345/99",
+            "06AB123456711"
         ]
         bad_urns = [
             "123",
-            "00bb/0000000/00",
```

Comment:
Why is 00bb/0000000/00 a bad URN?

They miss out the first slash in Staffordshire I think.